CMS-Enterprise/easi-app
# [EASI-4549] System workspace "Team" card

# EASI-4549
https://www.figma.com/design/z8GeZPDfIh9oKfHK1cX7Fo/System-centric-EASi?node-id=4239-8187&t=b9iFjrGAECM00YdC-0

## Description

- New `SystemWorkspace/TeamCard`
- Sort the team member roles by `teamRolesIndex`
- Lifted `Avatar` from Mint
- Button will link up in later tickets

## How to test this change

- Unit test snapshot with the rendered team list
- Seed and launch local dev
  - View a system in My Systems
    - Select a system to see the Workspace page with the new TeamCard

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
